src/Strategy/Python/Pip.hs

```diff
@@ -0,0 +1,126 @@
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+
+module Strategy.Python.Pip (Package (..), PackageMetadata (..), getPackages, pipShowParser) where
+
+import Control.Effect.Diagnostics (Diagnostics, recover)
+import Data.Aeson (FromJSON, ToJSON)
+import Data.Functor (void)
+import Data.List (find)
+import Data.Maybe (mapMaybe)
+import Data.String.Conversion (toText)
+import Data.Text (Text)
+import Data.Text qualified as Text
+import Data.Void (Void)
+import Effect.Exec (
+  AllowErr (Never),
+  Command (..),
+  Exec,
+  Has,
+  execJson,
+  execParser,
+ )
+import GHC.Generics (Generic)
+import Path (Abs, Dir, Path)
+import Text.Megaparsec
+import Text.Megaparsec.Char (char)
+import Text.Megaparsec.Char.Lexer qualified as L
+
+type Parser = Parsec Void Text
+data InstalledPackage = InstalledPackage
+  { name :: Text
+  , version :: Text
+  }
+  deriving (Show, Eq, Ord, Generic, FromJSON)
+
+data PackageMetadata = PackageMetadata Text Text [Text]
+  deriving (Show, Eq, Ord)
+
+data Package = Package
+  { pkgName :: Text
+  , pkgVersion :: Text
+  , requires :: [Package]
+  }
+  deriving (Show, Eq, Ord, Generic, FromJSON, ToJSON)
+
+pythonPip :: [Text] -> Command
+pythonPip args =
+  Command
+    { cmdName = "python"
```

Comment:
[Question] Lot's of older linux distro, have `python` defaulting to `python2`, and `python3` defaulting to `python3`. How do we address this scenario. For instance, if customer is using older distro, we will likely not report edges, since `python` will default to `python2`? We should either add this in FAQ. We should also provide workaround for our customers as well.

https://stackoverflow.com/questions/64801225/python-or-python3-what-is-the-difference

Reply:
I can't think of any other language ecosystem we support which has multiple versions of interpreter installed on system. I think adding docs should suffice. On worst case scenario, customers can modify path variable for fossa cmd.

Reply:
I believe the work around is to use a virtualenv. I've updated the docs to encourage customers to use a virtual environment. Since we're using `python -m pip` that should allow them to specify the python version in the virtual env.

Reply:
Oh and I did test this with python2.7, `pip list` and `pip show` do work with python2.7 so we should still report edges.